usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -289,6 +289,15 @@ def handle(self, *args, **options):
             award = load_data_into_model(Award(), row, field_map=award_field_map, value_map=award_value_map, as_dict=True)
             award, created = Award.objects.get_or_create(**award)
 
+            fad_value_map = {
+                "award": award,
+                "submission": submission_attributes
+
+            }
+
+            financial_assistance_data = load_data_into_model(FinancialAssistanceAward(), row, value_map=fad_value_map, as_dict=True)
+            fad, created = FinancialAssistanceAward.objects.get_or_create(**financial_assistance_data)
```

Comment:
This could potentially make more financial assistance award objects than we want because it's checking for a match on all of the fields. I think we might need a way to update a record if this matches the award ID and mod number, rather than creating a new record.